apps/examples/kernel_sample/mqueue.c

```diff
@@ -371,7 +371,7 @@ void mqueue_test(void)
 	status = pthread_cancel(receiver);
 	if (status == ESRCH) {
 		printf("mqueue_test: receiver has already terminated\n");
-		expected = (FAR void *)0;
+		expected = (FAR void *)PTHREAD_CANCELED;
```

Comment:
Then it is not necessary, you already expected that PTHREAD_CANCELED in line 370